### PR TITLE
CEDS-6171: Added page number to dashboard and saved-declaration page …

### PR DESCRIPTION
**Pull request by sunil-sethumadhavan**
…title check

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/DashboardStepDef.scala [hunk 1]
```diff
@@ -21,7 +21,7 @@ import uk.gov.hmrc.test.ui.pages.common.DashboardPage
 
 class DashboardStepDef extends BaseStepDef {
 
-  And("""^I should land on Dashboard page""")(() => DashboardPage.checkPage())
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/DashboardStepDef.scala:24]: You should not change `checkPage` to `checkDashboardPage`. This method override a method in `BasePage`.
 In addition, it is superfluous as due to the file name, `DashboardPage, we already now what page we are checking.

File: src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/SavedDeclarationsStepDef.scala [hunk 2]
```diff
@@ -21,7 +21,7 @@ import uk.gov.hmrc.test.ui.pages.common.SavedDeclarationsPage
 
 class SavedDeclarationsStepDef extends BaseStepDef {
 
-  And("""^I should land on Saved-Declarations page""")(() => SavedDeclarationsPage.checkPage())
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/cucumber/stepdefs/common/SavedDeclarationsStepDef.scala:24]: Same here

File: src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala [hunk 3]
```diff
@@ -137,4 +137,22 @@ object DashboardPage extends BasePage {
         println(s"Current Page: ${activePage.getText}")
       }
     }
-}
+
+  def checkDashboardPage(): Unit = {
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala:141]: Any reason to not call it `checkPage`?

File: src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala [hunk 4]
```diff
@@ -137,4 +137,22 @@ object DashboardPage extends BasePage {
...
+    checkUrlAndTitleDashboardPage()
+    checkBackButton()
+    checkPageLinks()
+    checkExpanders()
+  }
+
+  def checkUrlAndTitleDashboardPage(): Unit = {
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/common/DashboardPage.scala:148]: Why not `checkUrlAndTitle`? Repeat `DashboardPage` is superfluous.

File: src/test/scala/uk/gov/hmrc/test/ui/pages/common/SavedDeclarationsPage.scala [hunk 5]
```diff
@@ -52,4 +52,21 @@ object SavedDeclarationsPage extends BasePage {
 
   def clickDUCRLinkForAmendedDeclaration(): Unit = dUCRLink().click()
 
+  def checkSavedDeclarationsPage(): Unit = {
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/common/SavedDeclarationsPage.scala:55]: Same here

File: src/test/scala/uk/gov/hmrc/test/ui/pages/common/SavedDeclarationsPage.scala [hunk 6]
```diff
@@ -52,4 +52,21 @@ object SavedDeclarationsPage extends BasePage {
...
+    checkUrlAndTitleSavedDeclarationsPage()
+    checkBackButton()
+    checkPageLinks()
+    checkExpanders()
+  }
+
+  def checkUrlAndTitleSavedDeclarationsPage(): Unit = {
```
Comment (udinobi) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/common/SavedDeclarationsPage.scala:62]: Same here

**Review by udinobi — DISMISSED** [review]

**Review by dhanist-gadhiraju — APPROVED** [review]

**Pull request merged**